packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -1011,6 +1011,10 @@ oie.selfservice.unlock_user.landing.to.app.signing.in.message = 》Åççöûñ
 oie.selfservice.unlock_user.success.message = 》Ýöû çåñ ļöĝ îñ ûšîñĝ ýöûŕ éẋîšţîñĝ ûšéŕñåɱé åñð þåššŵöŕð· ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.selfservice.unlock_user.failed.message = 》Ŵé åŕé ûñåƀļé ţö ûñļöçķ ýöûŕ åççöûñţ åţ ţĥîš ţîɱé، þļéåšé çöñţåçţ ýöûŕ åðɱîñîšţŕåţöŕ ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.selfservice.user.unlock.not.allowed = 》Šéļƒ Šéŕṽîçé Ûñļöçķ îš ñöţ åļļöŵéð åţ ţĥîš ţîɱé· Þļéåšé çöñţåçţ šûþþöŕţ ƒöŕ åššîšţåñçé· ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.kmsi.title = 》Ķééþ ɱé šîĝñéð îñ 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.kmsi.subtitle = 》Šîĝñ îñ åñð éñţéŕ šéçûŕîţý ɱéţĥöðš ļéšš ƒŕéǫûéñţļý· 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.kmsi.accept = 》Šţåý šîĝñéð îñ โ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.kmsi.reject = 》Ðöñ´ţ šţåý šîĝñéð îñ €홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
do we merge in ok_PL files? I thought they were auto-generated? Maybe not for the SIW.

Reply:
yeah, we commit these changes in SIW